explorer/src/components/common/Copyable.tsx

```diff
@@ -1,56 +1,98 @@
-import React, { useState, ReactNode } from "react";
+import React, { ReactNode, useState } from "react";
 
-type CopyableProps = {
+type CopyState = "copy" | "copied" | "errored";
+
+export function Copyable({
+  text,
+  children,
+  replaceText,
+}: {
   text: string;
   children: ReactNode;
-  bottom?: boolean;
-  right?: boolean;
-};
+  replaceText?: boolean;
+}) {
+  const [state, setState] = useState<CopyState>("copy");
 
-type State = "hide" | "copy" | "copied";
+  const handleClick = async () => {
+    try {
+      await navigator.clipboard.writeText(text);
```

Comment:
note that this is now inside a try catch, your code wasn't catching the thrown error

Reply:
Oops, looks like I forgot something.. :)

Reply:
I think I missed a commit, but you covered it.